qucs-activefilter/CMakeLists.txt

```diff
@@ -44,30 +28,31 @@ CONFIGURE_FILE (
 
 INCLUDE_DIRECTORIES("${PROJECT_BINARY_DIR}")
 
-IF(WITH_QT6)
-	FIND_PACKAGE( Qt6 COMPONENTS Core Gui Widgets Svg SvgWidgets REQUIRED)
-
-INCLUDE_DIRECTORIES(
-      ${Qt6Core_INCLUDE_DIRS}
-      ${Qt6Widgets_INCLUDE_DIRS}
-      ${Qt6Svg_INCLUDE_DIRS}
-      ${Qt6SvgWidgets_INCLUDE_DIRS}
+find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Core Gui Widgets Svg SvgWidgets)
```

Comment:
Linux build with Qt5 fails on this line. Qt5 doesn't provide SvgWidgets package, But CMake tries to find it. Using CMake 3.18 on Ubuntu18.04.

Reply:
I think latest commit will solve this.

Reply:
Yes, the problem with SvgWidgets library has been resolved.